common-code/azure_wrappers/azure_container.py

```diff
@@ -0,0 +1,355 @@
+import logging
+import os
+import sys
+from functools import lru_cache
+from io import BytesIO, StringIO
+from logging import Logger
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+import PyPDF2
+from azure.core.exceptions import ResourceNotFoundError
+from azure.identity import (
+    AzureCliCredential,
+    ChainedTokenCredential,
+    DefaultAzureCredential,
+    EnvironmentCredential,
+    ManagedIdentityCredential,
+)
+from azure.keyvault.secrets import SecretClient
+from azure.storage.blob import (
+    BlobClient,
+    BlobServiceClient,
+    ContainerClient,
+    ContentSettings,
+)
+
+from azure_box import data_storage as ds
+from azure_box import version_info
+
+LOGGER = Logger(__file__)
+DEFAULT_TENANT_ID = "YOUR-TENANT-ID-HERE"
+# NOTE: where the code uses an environment variable e.g. tenant_id = os.environ.get("TENANT_ID", DEFAULT_TENANT_ID); we have
+# set that environment variable in our docker files.
+
+# reference: https://stackoverflow.com/questions/43878953/how-does-one-detect-if-one-is-running-within-a-docker-container-within-python
+SECRET_KEY = os.environ.get("AM_I_IN_A_DOCKER_CONTAINER", False)
+RUNNING_ON_WEBSITE = os.environ.get("WEBSITES_PORT", False)
```

Comment:
We can probably remove the RUNNING_ON_WEBSITE variable from this. We might be able to remove the SECRET_KEY as well, though I'm not sure if it's referenced elsewhere.

Reply:
```suggestion
```